src/main/java/com/commercetools/project/sync/customobject/CustomObjectSyncer.java

```diff
@@ -132,4 +143,30 @@ private static String getCustomObjectResourceIdentifier(
         .map(CustomObjectCompositeIdentifier::toString)
         .orElse("");
   }
+
+  private List<String> getExcludedContainerNames() {
```

Comment:
Can this method change from `private` to `default` access modifier, so that `CustomObjectSyncerTest` is not necessary to implement this method again.

Reply:
But when you test sth, then in the test verification you need to somehow get the result by yourself and not rely on the functions that you're testing? If I use one function and I'm testing it at the same time, when I change the implementation, the tests will still pass, even though it should fail.